Sources/FileSystem/Configuration/TestflightConfigurationProcessor.swift

```diff
@@ -54,4 +54,43 @@ struct TestflightConfigurationProcessor {
         }
     }
 
+    func readConfiguration() throws -> TestflightConfiguration {
+        let folder = try Folder(path: path)
+        var configuration = TestflightConfiguration()
+
+        let decodeBetaTesters: (Data) throws -> [BetaTester] = { data in
+            var configuration = CSVReader.Configuration()
+            configuration.headerStrategy = .firstLine
+
+            let csv = try CSVReader.decode(input: data, configuration: configuration)
+
+            return try csv.records.map { record in
+                try BetaTester(
+                    email: record[BetaTester.CodingKeys.email.rawValue],
+                    firstName: record[BetaTester.CodingKeys.firstName.rawValue],
+                    lastName: record[BetaTester.CodingKeys.lastName.rawValue]
+                )
+            }
+        }
+
+        configuration.appConfigurations = try folder.subfolders.map { appFolder in
+            let appYAML = try appFolder.file(named: "app.yml").readAsString()
```

Comment:
Common file names such as `"app.yml"` and `"beta-testers.csv"` will be extracted to constants to share between read and write